src/test/java/org/example/dao/MySqlJobRoleDaoTest.java

```diff
@@ -43,6 +50,12 @@ public void tearDown() throws Exception {
     @Test
     public void getJobRoles_shouldReturnListOfJobRoles_whenDatabaseReturnsRowsOfJobRoles() throws JobRoleDaoException,
             Exception {
+
+        try (Statement stmt = h2Connection.createStatement()) {
+            stmt.execute("DROP TABLE IF EXISTS roles");
+        }
```

Comment:
I would put this in the beforeeach before you start running the schema script and after you setup the h2Connection. Have a look at the docs, beforeeach runs before each test and saves duplicating the code. Don't think you need to wrap it in a try either

Reply:
fixed!